guiEditor/src/components/propertyTab/propertyGrids/gui/gridPropertyGridComponent.tsx

```diff
@@ -123,10 +127,10 @@ export class GridPropertyGridComponent extends React.Component<IGridPropertyGrid
 
     render() {
         const grid = this.props.grid;
-        const cols = [];
-
-        for (var index = 0; index < grid.rowCount; index++) {
-            cols.push(grid.getColumnDefinition(index));
+        if (grid !== this._previousGrid) {

```

Comment:
this is usually solved better with the react state instead of local variable. have you tried that?